crates/stc_ts_file_analyzer/src/analyzer/assign/mod.rs

```diff
@@ -2808,13 +2808,94 @@ impl Analyzer<'_, '_> {
     /// Should be called only if `to` is not expandable.
     fn assign_to_intrinsic(&mut self, data: &mut AssignData, to: &StringMapping, r: &Type, opts: AssignOpts) -> VResult<()> {
         match to.kind {
-            IntrinsicKind::Uppercase => {}
-            IntrinsicKind::Lowercase => {}
-            IntrinsicKind::Capitalize => {}
-            IntrinsicKind::Uncapitalize => {}
+            IntrinsicKind::Uppercase => {
```

Comment:
Generally I think the code is overly verbose for what's its doing, any suggestions?